build/default.ps1

```diff
@@ -62,10 +62,12 @@ task Test {
 
     # TODO: Move to using dotnet test to execute the unit tests
```

Comment:
Is this TODO still relevant? Can it be removed?

Reply:
I'm going to leave this for now since its unrelated to the PR, I will need to properly check the `dotnet test` output vs the `nunit` cli output.